### PR TITLE
[Merged by Bors] - Minor docs edit/add in bevy_render

### PR DESCRIPTION
**Pull request by djeedai**
# Objective

Docs updates.

## Solution

- Detail what `OrthographicCameraBundle::new_2d()` creates.
- Fix a few renamed parameters in comments of `TrackedRenderPass`.
- Add missing comments for viewport and debug markers.

### CODE REVIEW

File: crates/bevy_render/src/camera/bundle.rs
Comment (alice-i-cecile) [inline crates/bevy_render/src/camera/bundle.rs:85]: > The scaling mode is the default for

As a reader, I'm not immediately sure what this refers to. Can this be configured somewhere?

File: crates/bevy_render/src/camera/bundle.rs
Comment (alice-i-cecile) [inline crates/bevy_render/src/camera/bundle.rs:93]:
```suggestion
    /// camera) in world space.
```

File: crates/bevy_render/src/render_phase/draw_state.rs
Comment (alice-i-cecile) [inline crates/bevy_render/src/render_phase/draw_state.rs:279]: I would like to see a simple doc example here, or more breadcrumbs to how this might be used.

Comment (alice-i-cecile) [reply crates/bevy_render/src/render_phase/draw_state.rs:279]: Looking a bit more, I think that the debugging advice should actually be described in more depth as part of the `TrackedRenderPass` struct that owns these methods. It's not essential for this PR though.

**Comment by alice-i-cecile** [comment]
Looks good, just a few nits :)

File: crates/bevy_render/src/camera/bundle.rs
Comment (djeedai) [reply crates/bevy_render/src/camera/bundle.rs]: Fixed.

File: crates/bevy_render/src/camera/bundle.rs
Comment (djeedai) [reply crates/bevy_render/src/camera/bundle.rs]: ortho

File: crates/bevy_render/src/render_phase/draw_state.rs
Comment (djeedai) [reply crates/bevy_render/src/render_phase/draw_state.rs:279]: debug

**Review by alice-i-cecile — APPROVED** [review]

**Comment by djeedai** [comment]
Ready for merge I think, likely not controversial so ping @mockersf :) thanks!

File: crates/bevy_render/src/camera/bundle.rs
Comment (mockersf) [inline crates/bevy_render/src/camera/bundle.rs:87]: Is it interesting to mention that "pixels" is logical pixels? More and more screen have a scale factor

Comment (djeedai) [reply crates/bevy_render/src/camera/bundle.rs]: Added.

File: crates/bevy_render/src/camera/bundle.rs
Comment (superdump) [inline crates/bevy_render/src/camera/bundle.rs:88]: Hmm, I haven't checked but the range [-960, 960] inclusively would be 1921 pixels. I wonder if for a dimension divisible by 2, the pixel coordinates are that their centres and so are in the range [-959.5, 959.5] inclusively and the centres in the x dimension are at every .5 value. Do you or does anyone know?

File: crates/bevy_render/src/camera/bundle.rs
Comment (superdump) [inline crates/bevy_render/src/camera/bundle.rs:92]: 0.0 to -1000.0 in camera view space as the projection has `near` = 0.0 and `far` = 1000.0 and is right-handed y-up. `near` and `far` refer to parameters of the projection as to the distance to the closest and farthest objects that will be rendered by the projection. Right-handed y-up still means that positive z points out of the screen, so 1000.0 units in front of the camera is at z = -1000.0 in camera view space. Hopefully that was a clear elaboration but let me know if something was confusing or incorrect. :)

File: crates/bevy_render/src/camera/bundle.rs
Comment (inodentry) [inline crates/bevy_render/src/camera/bundle.rs:95]: Since you are adding docs for `new_2d`, you should probably also add some similar docs for `new_3d` below? It feels weird to leave it without docs.

The 3D ortho projection has the same right-handed coordinate system, but the default scaling mode is FixedVertical, spanning -/+ `-scale` along the vertical axis and -/+ `scale * aspect_ratio` along the horizontal. There is also no Z positioning weirdness.

File: crates/bevy_render/src/camera/bundle.rs
Comment (djeedai) [reply crates/bevy_render/src/camera/bundle.rs:88]: I think possibly the pixel center is not aligned with the coordinates, but the pixel corners are. In any case, I didn't feel like digging into those details; this can be fixed once merged if someone knows the exact formula. I'd rather merge _some_ form of documentation soon since there's absolutely nothing, and some other parts contain important information that need to be documented.

File: crates/bevy_render/src/camera/bundle.rs
Comment (djeedai) [reply crates/bevy_render/src/camera/bundle.rs:92]: Right, "Z is inverted".

File: crates/bevy_render/src/camera/bundle.rs
Comment (djeedai) [reply crates/bevy_render/src/camera/bundle.rs:95]: I have another PR ready which builds over this and adds docs to the entire `bevy_render::camera` module subtree, but touches the same files, so I'm waiting for this to merge first.

File: crates/bevy_render/src/camera/bundle.rs
Comment (djeedai) [reply crates/bevy_render/src/camera/bundle.rs:92]: > the distance to the closest and farthest objects

Note that I explicitly avoided that phrasing because it's not correct; clipping occurs at the fragment level in shaders, not object level. There may be CPU-side clipping too at object level, but technically an object at distance Z=-999.99 may be partially invisible if it extends beyond 1000.0 units from the camera, so it's not a boolean show/hide at the object level.

File: crates/bevy_render/src/camera/bundle.rs
Comment (djeedai) [reply crates/bevy_render/src/camera/bundle.rs:88]: In any case, the phrasing "coordinates visible on screen" is already vague in what it refers to, so I don't feel there's a need to be more precise here. I just wanted to carry the general idea. Or, I'm happy to replace it if someone wants to suggest a better phrasing for the entire sentence.

File: crates/bevy_render/src/camera/bundle.rs
Comment (djeedai) [reply crates/bevy_render/src/camera/bundle.rs]: Fixed.

File: crates/bevy_render/src/camera/bundle.rs
Comment (superdump) [reply crates/bevy_render/src/camera/bundle.rs:88]: I guess we can roll with this and then correct it later.

**Pull request closed without merging**
